### PR TITLE
Added back fis ds update application

### PR DESCRIPTION
**Pull request by yogendraU**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###
Add FIS DS Update application for DSS Update Case application

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: environments/stg/stg.tfvars [hunk 1]
```diff
@@ -1664,5 +1664,20 @@ frontends = [
     custom_domain  = "em-icp.aat.platform.hmcts.net"
     backend_domain = ["firewall-prod-int-palo-cftaat.uksouth.cloudapp.azure.com"]
 
+  },
+  {
+    product          = "dss-update-case"
+    name             = "dss-update-case"
+    custom_domain    = "dss-update-case.aat.platform.hmcts.net"
+    backend_domain   = ["firewall-nonprodi-palo-cftaatappgateway.uksouth.cloudapp.azure.com"]
```
Comment (reespozzi) [inline environments/stg/stg.tfvars:1672]: I think this should be `"firewall-prod-int-palo-cftaat.uksouth.cloudapp.azure.com"`

File: environments/ithc/ithc.tfvars [hunk 2]
```diff
@@ -1154,5 +1154,20 @@ frontends = [
     custom_domain    = "et-sya.ithc.platform.hmcts.net"
     backend_domain   = ["firewall-nonprodi-palo-cftithc.uksouth.cloudapp.azure.com"]
     certificate_name = "wildcard-ithc-platform-hmcts-net"
+  },
+  {
+    product          = "dss-update-case"
+    name             = "dss-update-case"
+    custom_domain    = "dss-update-case.ithc.platform.hmcts.net"
+    backend_domain   = ["firewall-nonprodi-palo-cftithcappgateway.uksouth.cloudapp.azure.com"]
```
Comment (reespozzi) [inline environments/ithc/ithc.tfvars:1162]: I think this should be `"firewall-nonprodi-palo-cftithc.uksouth.cloudapp.azure.com"`

File: environments/test/test.tfvars [hunk 3]
```diff
@@ -1318,5 +1318,20 @@ frontends = [
     custom_domain    = "et-sya.perftest.platform.hmcts.net"
     backend_domain   = ["firewall-nonprodi-palo-cft-perftest.uksouth.cloudapp.azure.com"]
     certificate_name = "wildcard-perftest-platform-hmcts-net"
+  },
+  {
+    product          = "dss-update-case"
+    name             = "dss-update-case"
+    custom_domain    = "dss-update-case.perftest.platform.hmcts.net"
+    backend_domain   = ["firewall-nonprodi-palo-cftperftestappgateway.uksouth.cloudapp.azure.com"]
```
Comment (reespozzi) [inline environments/test/test.tfvars:1326]: This should be `"firewall-nonprodi-palo-cft-perftest.uksouth.cloudapp.azure.com"`

**Review by reespozzi — CHANGES_REQUESTED** [review]

File: environments/stg/stg.tfvars (hunk 1 above)
Comment (yogendraU) [reply environments/stg/stg.tfvars]: thanks @reespozzi , I have done the changes

File: environments/ithc/ithc.tfvars (hunk 2 above)
Comment (yogendraU) [reply environments/ithc/ithc.tfvars]: thanks @reespozzi , I have done the changes

File: environments/test/test.tfvars (hunk 3 above)
Comment (yogendraU) [reply environments/test/test.tfvars]: thanks @reespozzi , I have done the changes

**Pull request merged**